include/cutlass/detail/helper_macros.hpp

```diff
@@ -156,6 +159,16 @@ namespace cutlass {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
+#if defined(CUTLASS_ENABLE_SYCL)
+#if defined(__SYCL_DEVICE_ONLY__) && defined(__NVPTX__)
+#  define ENABLE_NVPTX 1
+#endif
+#else
+#  define ENABLE_NVPTX 1
+#endif
```

Comment:
can we have this definition in one place, with the comments

Reply:
I'm not sure why but Cutlass redefine all these generic macros in cutlass and cute.

Reply:
yeah I dont know either, probably because of the legacy support. Lets keep it as is